src/types/comments.ts

```diff
@@ -0,0 +1,15 @@
+export interface CommentProps {
+  siteName: string
+  requestId: number
+}
+
+export type UpdateCommentProps = CommentProps & {
+  message: string
+}
+
+export interface CommentData {
+  message: string
+  createdAt: number
```

Comment:
hmm, I assume this is going to be of type `DateType` right? shouldnt we use `string` instead to parse the time from the backend?

Reply:
Hmm @seaerchin has proposed returning `createdAt` in epoch time instead - so the backend converts the string format retrieved by Github into a number!